.github/workflows/OCV-PR-4.x-W10.yaml

```diff
@@ -172,9 +172,8 @@ jobs:
     - name: Accuracy:videoio
       timeout-minutes: 60
       if: ${{ always() && steps.build-opencv.outcome == 'success' }}
-      run: |
-        # batch script is used to run videoio accuracy test iteratively in case of random failures in a parallel run on a self-hosted runner
-        cd ${{ github.workspace }}\build && %CI_SCRIPTS%\videoio.bat
+      # batch script is used to run videoio accuracy test iteratively in case of random failures in a parallel run on a self-hosted runner
+      run: cd ${{ github.workspace }}\build && %CI_SCRIPTS%\videoio.bat
```

Comment:
The issue is with videoio.bat script - it uses GHA rendering string `${{ ... }}` inside.